src/System.IO.FileSystem/src/System/IO/CharSpanExtensions.Unix.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace System.IO
+{
+    internal static partial class CharSpanExtensions
+    {
+        internal static unsafe bool EqualsOrdinal(ReadOnlySpan<char> first, ReadOnlySpan<char> second, bool ignoreCase = false)
+        {
+            if (first.Length != second.Length)
+                return false;
+
+            if (ignoreCase == true)
+                return first.SequenceEqual(second);
```

Comment:
I don't understand the logic here.  Shouldn't this be `if (!ignoreCase)`?

Reply:
Yes, my mistake.

Reply:
also `==true` and `==false` are ugly in general.